.github/workflows/ci.yaml

```diff
@@ -34,7 +34,22 @@ jobs:
       - name: Sign the binary with EGo for production
         run: make sign-prod
 
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+        id: go
+
+      - name: Checkout code into the Go module directory
+        uses: actions/checkout@v3
+
       - name: Lint
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.30
+          version: latest
+          args: --timeout=5m
```

Comment:
Since the `golangci-lint` eats lot of memory, I'm gonna run this job on Ubuntu hosted by GitHub, not self-hosted.